remoteappmanager/static/js/gamodule.js

```diff
@@ -0,0 +1,26 @@
+// This module contains the setup for google analytics.
+// MUST not be renamed to analytics. Some blockers rely on name 
+// matching to prevent loading.
+define([
+], function () {
+    "use strict";
+
+    function init() {
+        var module;
+        
+        if (window.apidata.analytics !== undefined) {
+            window.ga('create', window.apidata.analytics.tracking_id, 'auto');
+        } else {
+            window.ga = function () {
```

Comment:
I think `window.ga = undefined;` and `window.ga = function () {};` will have the same effect but for the second option jslint would complain about empty brackets